### DISCUSSION TITLE
how to use Multi-Tab & fzf, zoxide

### DISCUSSION THREAD

**Discussion by fastfading** [sxyazi/yazi#859] [1. Q&A]
there are many feature such as Multi-Tab & fzf, zoxide
mentioned in https://yazi-rs.github.io/features
however there is no doccument for it .

new users like me are very confused.
could you tell me how to use Multi-Tab & fzf, zoxide ?

how to create / delete new tab
how to use fzf (Z), zoxide(z)
I have install zoxide , but got error "zoxide: no match found" when I hit z button

since copy cross multiple yazi is not supported, this multi-tab is very useful.

**Comment by romanskorobagatko** [comment]
"I have install zoxide , but got error "zoxide: no match found" when I hit z button" - same here, would be great to have a short tutorial on how to configure it.

**Comment by fastfading** [comment]
[yazi/yazi-config/preset/keymap.toml at main · sxyazi/yazi · GitHub](https://github.com/sxyazi/yazi/blob/main/yazi-config/preset/keymap.toml)



[GitHub - ajeetdsouza/zoxide: A smarter cd command. Supports all major shells.](https://github.com/ajeetdsouza/zoxide)

```shell
brew install zoxide
vi ~/.zshrc  
# add following at the end
eval "$(zoxide init zsh)"

zoxide import --from=z $HOME/.z

restart terminal
yazi
hit z button

```

**Comment by sxyazi** [comment]
Hi, zoxide will be reimplemented as a plugin in the next version of Yazi, which will solve this problem

**Reply by ccoVeille** [reply]
Issue is here for anyone interested https://github.com/sxyazi/yazi/issues/865

**Reply by sxyazi** [reply]
Done by https://github.com/sxyazi/yazi/issues/865
